documentation/asciidoc/services/connect/troubleshooting.adoc

```diff
@@ -10,7 +10,7 @@
 
 * Browsers intercept certain keys and key combinations. As a result, you can't type these keys into your Connect window. Screen sharing includes a toolbar to simulate some of the most popular intercepted keys.
 
-* Upgrading `rpi-connect` and `rpi-connect-lite` using Connect is not supported. The upgrade process will terminate all remote shell sessions and drop all connections.
+* Upgrading `rpi-connect` and `rpi-connect-lite` using Connect's remote shell is not supported. The upgrade process will terminate all remote shell sessions and drop all connections. To upgrade in a remote shell session the upgrade process must be launched using a tool like `screen` to ensure the process continues uninterrupted after the session is terminated.
```

Comment:
"To upgrade in a" -> "To upgrade Connect in a" ?

"after the session is terminated." -> "after the parent session is terminated." ?